src/test/java/seedu/address/logic/parser/TutorsPetParserTest.java

```diff
@@ -181,6 +187,20 @@ public void parseCommand_deleteLesson() throws Exception {
                 + PREFIX_CLASS_INDEX + INDEX_FIRST_ITEM.getOneBased()), command);
     }
 
+    @Test
+    public void parseCommand_addAttendance() throws Exception {
+        Week week = new Week(INDEX_FIRST_ITEM);
+        Attendance attendance = new Attendance(VALID_PARTICIPATION_SCORE_80);
+        AddAttendanceCommand command =
+                new AddAttendanceCommand(INDEX_FIRST_ITEM, INDEX_FIRST_ITEM, INDEX_FIRST_ITEM, week, attendance);
+        assertEquals(parser.parseCommand(AddAttendanceCommand.COMMAND_WORD + " "
+                + PREFIX_CLASS_INDEX + INDEX_FIRST_ITEM.getOneBased() + " "
+                + PREFIX_LESSON_INDEX + INDEX_FIRST_ITEM.getOneBased() + " "
+                + PREFIX_STUDENT_INDEX + INDEX_FIRST_ITEM.getOneBased() + " "
+                + PREFIX_WEEK + INDEX_FIRST_ITEM.getOneBased() + " "
+                + PREFIX_PARTICIPATION_SCORE + "80 "), command);
```

Comment:
It seems like the expected vs actual fields are swapped? It might be good to also declare the parser version as the `command` and the `AddAttendanceCommand` version as `expectedCommand`?